ccd-definition/CaseEvent/User/UserEvents-SDO-nonprod.json

```diff
@@ -15,5 +15,23 @@
     "EndButtonLabel": "Submit",
     "RetriesTimeoutAboutToStartEvent": 0,
     "RetriesTimeoutURLAboutToSubmitEvent": 0
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "NotSuitable_SDO",
+    "Name": "Not suitable for SDO",
+    "Description": "Not suitable for SDO",
+    "DisplayOrder": 18,
+    "PreConditionState(s)": "*",
```

Comment:
I believe the precondition needs to be JUDICIAL_REFERRAL and postcondition needs to be PROCEED_TO_HERITAGE_SYSTEM

Reply:
+1